index.html

```diff
@@ -5,55 +5,108 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,500,700,900&subset=latin,cyrillic">
+    <link rel="stylesheet" href="style.css">
     <title>Gllacy</title>
 </head>
 
+
 <body>
-    <input type="radio">
-    <input type="radio">
-    <input type="radio">
-    <!--Доверстать радио кнопки, дать спили, найти спозоб сделать из них слайдер заднего фона. -->
+
+
+    <input class="visibility-hidden" type="radio" id="fon-1" name="toggle" checked>
+    <input class="visibility-hidden" type="radio" id="fon-2" name="toggle">
```

Comment:
```suggestion
    <input class="visibility-hidden" type="radio" id="slider__radio-1" name="toggle">
```
какой нахуй транслит???

Reply:
так же и с первым и с третьим